light-zk.js/src/idls/verifier_program_one.ts

```diff
@@ -133,13 +133,13 @@ export type VerifierProgramOne = {
           "isSigner": false
         },
         {
-          "name": "verifierState",
+          "name": "eventMerkleTree",
           "isMut": true,
           "isSigner": false
         },
         {
-          "name": "eventMerkleTree",
-          "isMut": true,
+          "name": "verifierState",
+          "isMut": false,
```

Comment:
Do tests run with verifier state being not mutable?

Reply:
It's fixed now.